datasets/modeled/transport_stock/dimension_mappings/scenario_to_scenario.csv

```diff
@@ -0,0 +1,5 @@
+from_id,to_id
+ief_breakthrough,breakthrough
+ief_high,high
+ief_high_inefficient_electrification,inefficient
```

Comment:
@ashreeta, maybe our `inefficient` scenario should instead be `high_inefficient`?